src/api/specimen/ScheduleSpecimenMAS.ts

```diff
@@ -32,6 +29,8 @@ const ScheduleSpecimenMAS = async (handle: string, MASRequest: Dict, batching: b
             specimenMAS = data.data[0].attributes;
         } catch (error) {
             console.warn(error);
+
```

Comment:
remove space

Reply:
You mean the enter? That is intentional

Reply:
alright well **I** would remove it :laughing: java conventions are different i guess